sporkfish/searcher/minimax.py

```diff
@@ -64,6 +65,7 @@ def __init__(
             logging.info("Disabled transposition table in search.")
 
         self._evaluator = evaluator
+        self.max_depth = searcher_config.max_depth
```

Comment:
Is this needed? I thought this would already be stored somewhere (I might be wrong, looking on the phone). If we do need it can we make it protected?

Reply:
The max depth parameter, I thought it would be be passed somewhere alr

Reply:
All clear - not sure why the github review interface made it look it you commented on a blank line before...Looks fine now!

Reply:
ah i could us _searcher_config.max_depth?

Reply:
No objections to adding one if it's not already there, but please make protected!